src/routes/reports.rs

```diff
@@ -64,32 +66,64 @@ pub async fn report_create(
     };
 
     match new_report.item_type {
-        ItemType::Project => {
-            report.project_id = Some(
-                serde_json::from_str::<ProjectId>(&format!(
-                    "\"{}\"",
-                    new_report.item_id
-                ))?
-                .into(),
+        ItemType::Project | ItemType::Mod => {
+            let project_id =
+                ProjectId(parse_base62(new_report.item_id.as_str())?);
```

Comment:
Why even wrap it in `ProjectId`? I would just pass this straight to the DB call. All ids are identical (they are just wrappers of a u64). Same w/ everything below

Reply:
Addressed on Discord